File: _examples/java/tuples.java
// Java 没有内置元组，使用 Record (Java 16+)
record Person(int age, String name, double height) {}
var person = new Person(25, "Alice", 1.68);

// 访问
int age = person.age();
String name = person.name();

// 或使用 Map.entry 作为二元组
var pair = Map.entry("key", 100);

File: _examples/ruby/tuples.rb
# Ruby 没有原生元组，使用数组或 Struct
person = [25, "Alice", 1.68].freeze

# 解构
age, name, height = person

# 使用 Struct 创建命名元组
Person = Struct.new(:age, :name, :height)
p = Person.new(25, "Alice", 1.68)
puts p.name